saleor/dashboard/forms.py

```diff
@@ -33,24 +43,58 @@ def valid_value(self, value):
         forms.Field.validate(self, value)
         return True
 
-    def set_initial(self, obj):
+    def set_initial(self, obj, obj_id=None, label=None):
         """Set initially selected objects on field's widget."""
-        selected = {'id': obj.pk, 'text': str(obj)}
+        selected = {
+            'id': obj_id if obj else obj.pk,
```

Comment:
I think this was supposed to say `obj_id if obj_id ...`?

Reply:
Yes, I'll fix it.